insights/combiners/satellite_version.py

```diff
@@ -95,30 +104,33 @@ def satellite_version(local, shared):
     information.
 
     """
-    # For Satellite 6.1.x (satellite_version/version.rb is available)
+    # For Satellite 6.1.x, if satellite_version/version.rb is available:
     sat6_ver = shared.get(Sat6Ver)
     if sat6_ver and sat6_ver.version:
         return SatelliteVersion(sat6_ver.full, sat6_ver.version, None,
                                 sat6_ver.major, sat6_ver.minor)
 
     rpms = shared.get(InstalledRpms)
     if rpms:
-        # For Satellite 6.2.x
+        # For Satellite 6.2.x, check the ``satellite`` package directly
         sat62_pkg = rpms.get_max('satellite')
         if sat62_pkg:
             version = sat62_pkg.version
             major, minor = _parse_sat_versoin(version)
             return SatelliteVersion(sat62_pkg.package, version,
                                     sat62_pkg.release, major, minor)
 
-        # For Satellite 6.0.x/6.1.x
+        # For Satellite 6.0.x/6.1.x, a little complex
```

Comment:
"a liitel complex" as comment is not good. you should give words to explain the codes.